src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java

```diff
@@ -34,24 +36,30 @@ public PaymentsPublisher(
 
     @Override
     public void send(CreatePaymentsCommand cmd) {
-        try {
-            String messageBody = objectMapper.writeValueAsString(cmd);
+        send(cmd, Labels.CREATE);
+    }
 
+    @Override
+    public void send(UpdatePaymentsCommand cmd) {
+        send(cmd, Labels.UPDATE);
+    }
+
+    private void send(Object cmd, String label) {
+        try {
             IMessage message = new Message(
-                cmd.ccdReference,
-                messageBody,
+                UUID.randomUUID().toString(),
+                objectMapper.writeValueAsString(cmd),
                 APPLICATION_JSON.toString()
```

Comment:
just out of nowhere question: is that a type of message saying it's gonna be json format and not byte array? i believe it goes into different parameter in service bus' message body object. i might be panicking in vain but - just asking

Reply:
Afaik it's just a regular property.
We never use it on the reading end btw.

Reply:
Down the line, theoretically, content type is there to distinguish what kind of body one is sending. ATM it still translates to byte array but in my opinion it's lack of restriction or implementation of service bus.

All good. Apparently was panicking too much as it ~looks like~ is translating to byte array anyway